assets/packages/themeoptions-core/themeoptions-core.js

```diff
@@ -72,6 +72,18 @@ var ThemeOptions = function () {
             }
 
             optionObject[$(item).attr('name')] = $(item).val();
+
+            // disable the preview image button if the image
+            // selected could not be mapped to one of the images
+            // that actually exists within the theme
+            const src = $(item).find('option:selected').data('lightbox-src');
+            const missing = src === '';
+            const itemId = $(item).attr('id');
+            const button = $(`button[data-bs-target="#${itemId}"]`);
+            button.prop('disabled', missing);
+
+            // add some feedback to the user, mark field invalid
+            $(item).toggleClass('is-invalid', missing);
```

Comment:
Thank you for  the review. Good catch, sorry I didn't notice it myself.

I've made it so that only image dropdowns get the data-lightbox-src attribute on the inherited option.

Also, changed the js to use the image dropdown specific class instead of the one that all dropdowns have.

I still haven't gotten the static analysis tools setup. Sorry if there are issues! If you let me know, I can fix them. If this PR is okay now, let me know and I will update 5.x PR also.

Also, I'm assuming whoever merges this will squash the commits. I was using force push to keep the same commit message, but I apparently forgot when I updated this pr previously. If you prefer me to fix the commit messages, I can figure something out.

Reply:
looks fine now :). Tests are successful, so code style is fine. When it gets merged, it will be done with a squash merge and the person will also update the commit message accordingly if it doesn't fit with the ticket.